### DISCUSSION TITLE
Docker guide?

### DISCUSSION THREAD

**Discussion by Reshaflame** [oobabooga/text-generation-webui#946] [Q&A]
I'm using a docker to run the SD auto1111 and it works great! 

I had a guide on how to install the docker and the webui on it - sort of "a guide for dummies" since I'm not a computer genius.. can someone please make a similar guide for this webui docker installation too?

**Comment by Zebradil** [comment]
https://github.com/oobabooga/text-generation-webui/wiki/Docker-compose-installation
